UI/ControlMappingScreen.h

```diff
@@ -78,27 +78,26 @@ class KeyMappingNewKeyDialog : public PopupScreen {
 	bool mapped_;  // Prevent double registrations
 };
 
-class KeyMappingNewMouseKeyDialog : public PopupScreen {
+class BindingChoice;
+
+class KeyMappingDialog : public PopupScreen {
 public:
-	explicit KeyMappingNewMouseKeyDialog(int btn, bool replace, std::function<void(KeyDef)> callback, std::shared_ptr<I18NCategory> i18n)
-		: PopupScreen(i18n->T("Map Mouse"), "", ""), callback_(callback), mapped_(false) {
-		pspBtn_ = btn;
-	}
+	explicit KeyMappingDialog(int btn, std::shared_ptr<I18NCategory> i18n)
+		: PopupScreen(std::string(i18n->T("Mapping for"))+" "+i18n->T(KeyMap::GetPspButtonName(btn))), pspBtn_(btn) {}
 
-	bool key(const KeyInput &key) override;
-	bool axis(const AxisInput &axis) override;
+	virtual bool key(const KeyInput &key) override;
+	virtual bool axis(const AxisInput &axis) override;
 
 protected:
 	void CreatePopupContents(UI::ViewGroup *parent) override;
 
-	bool FillVertical() const override { return false; }
-	bool ShowButtons() const override { return true; }
-	void OnCompleted(DialogResult result) override {}
+	virtual bool FillVertical() const override { return false; }
+	virtual bool ShowButtons() const override { return true; }
```

Comment:
nit: This is the default anyway?  Also, you don't actually show these buttons because the button1_ text is always "".  So this override isn't needed.

I suspect you copied this from elsewhere, but we're currently preferring overrides to be without the `virtual` (since it's already an `override`, it's duplicative to say `virtual` too.)

-[Unknown]

Reply:
Yeah was copy pasted form the previous mapping popup views (fixed them as well).